.github/workflows/release.yml

```diff
@@ -185,6 +185,100 @@ jobs:
           inlineScript: |
             az storage blob upload-batch --account-name ${{ env.PREFIX }}public -d '$web' -s frontend/dist
 
+  deploy_release_test:
+    name: "Deploy Release: TEST"
+    needs:
+      - build_release
+      - build_frontend_release
+    if: github.ref == 'refs/heads/master'
+    environment: test
+    concurrency: test
+    runs-on: ubuntu-latest
+    env:
+      RESOURCE_GROUP: prime-data-hub-test
+      ACR_REPO: pdhtestcontainerregistry.azurecr.io
```

Comment:
Just a note and something for longer term: I _know_ the format here is `pdh${ENVIRONMENT}containerregistry.azurecr.io`, but it's _super_-easy to read over that. Maybe for longer term: make the environment stand out a bit more...

Reply:
How would you suggest? This is a resource there are legacy naming limitations (i.e. only alphanumeric).

Reply:
I don't know yet :(